pkg/test/istioio/framework.go

```diff
@@ -184,14 +193,18 @@ func NeedSetup(config string) bool {
 // TestDocs traverses through all test cases and runs those that need the
 // setup config specified by the input. The (*testing.T) variable comes
 // from the TestDocs function in each tests/setup/*/doc_test.go
-func TestDocs(t *testing.T, config string) {
+func TestDocs(t *testing.T, config string, multicluster bool) {
```

Comment:
I don't think we want to fish `multicluster` through the test framework.  If we assume that we have a separate prow job for the multicluster tests, a better approach might be to just have that job select the specific tests to run via `TEST` (see https://github.com/istio/istio.io/tree/master/tests).

Reply:
Anyways this will be moved to another PR.  But I have a few questions
* Are we planning to put multicluster tests into a single directory? (Unlike what we do for single cluster tests)
* If not, then wouldn't we end up updating prow job config for new tests? (like `TEST=dir1` and then `TEST=dir1,dir2`)

Reply:
Alternatively, make file can be edited to avoid the issue mentioned in second point. But still a hassle.

Reply:
> Are we planning to put multicluster tests into a single directory? (Unlike what we do for single cluster tests)

Yes, the multicluster tests will follow the same directory structure as the site content, so the prow target would use something like `TEST=setup/install/multicluster`